test/test_script.py

```diff
@@ -990,9 +1045,9 @@ def test_cmd_datetime(self):
             self.assertScriptResultEquals("$datetime(%foo%)", "20200102123456.000789", context)
             self.assertScriptResultEquals("$datetime($initials())", "2020-01-02 12:34:56", context)
             # Tests with static input
-            self.assertScriptResultEquals("$datetime(\%Y\%m\%d\%H\%M\%S.\%f)", "20200102123456.000789", context)
+            self.assertScriptResultEquals("$datetime(\\%Y\\%m\\%d\\%H\\%M\\%S.\\%f)", "20200102123456.000789", context)
             # Tests with timezones
-            self.assertScriptResultEquals("$datetime(\%H\%M\%S \%z \%Z)", "123456 +0200 TZ Test", context)
+            self.assertScriptResultEquals("$datetime(\\%H\\%M\\%S \\%z \\%Z)", "123456 +0200 TZ Test", context)
```

Comment:
Maybe better use `r"$datetime(\%H\%M\%S \%z \%Z)"` instead, same above.

Reply:
Done in ca148ee